packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.html

```diff
@@ -477,122 +477,23 @@
 
         <div class="col-md-4">
             <!-- SUBSCRIPTION -->
-            <div class="oui-tile mb-5 h-100">
-                <h4
-                    class="oui-tile__title oui-heading_4"
-                    data-translate="server_subscription"
-                ></h4>
-                <div class="oui-tile__body">
-                    <div class="oui-tile__item">
-                        <dl class="oui-tile__definition">
-                            <dt class="oui-tile__term">
-                                <service-expiration-label
-                                    data-service-infos="$ctrl.serviceInfos"
-                                >
-                                </service-expiration-label>
-                            </dt>
-                            <dd class="oui-tile__description">
-                                <service-expiration-date
-                                    data-hide-renew-action="!$ctrl.server.canOrder"
-                                    data-service-name="$ctrl.server.name"
-                                    data-service-type="'DEDICATED_SERVER'"
-                                    data-service-infos="$ctrl.serviceInfos"
-                                    data-force-hide-renew-action="true"
-                                >
-                                </service-expiration-date>
-                            </dd>
-                        </dl>
-                        <div class="oui-tile__actions">
-                            <service-expiration-date
-                                data-hide-renew-date="true"
-                                data-hide-renew-action="!$ctrl.server.canOrder"
-                                data-service-name="$ctrl.server.name"
-                                data-service-type="'DEDICATED_SERVER'"
-                                data-service-infos="$ctrl.serviceInfos"
-                            >
-                            </service-expiration-date>
-                        </div>
-                    </div>
-                    <div
-                        class="oui-tile__item"
-                        data-ng-if="$ctrl.worldPart !== 'US'"
+            <billing-subscription-tile
+                service-path="{{ '/dedicated/server/' + $ctrl.server.name }}"
+                with-engagement
+            >
+                <billing-contact-actions>
+                    <oui-action-menu-item
+                        data-href="{{:: $ctrl.changeOwnerUrl }}"
```

Comment:
can we put this in the tile component? this feature is seen in many sservice dashboard.

Reply:
Here the display looks specific to the server state and I'm not sure all services can have owner change (I'm not sure it is available for VPS for instance)